tests/postgres_tests/test_json.py

```diff
@@ -19,6 +19,16 @@
     pass
 
 
+class TestModelMetaOrdering(PostgreSQLTestCase):
+
+    def test_ordering_by_json_field_value(self):
+        class TestJSONModel(JSONModel):
+            class Meta:
+                ordering = ['field__value', ]
+
+        self.assertFalse(TestJSONModel.check())
```

Comment:
To match other model `check()` tests, use `self.assertEqual(TestJSONModel.check(), [])`